samples/msal-node-samples/ElectronSystemBrowserTestApp/redirect/public/JS/handleRedirect.js

```diff
@@ -0,0 +1,17 @@
+"use static";
+
+import auth from "../config/authConfig.json" assert { type: "json" };
+
+window.addEventListener("DOMContentLoaded", function () {
+    let redirectUri = `${auth.customURLScheme}`;
```

Comment:
Why are you using a template literal here?

Reply:
Good catch removed template literal